src/formats/notion.ts

```diff
@@ -102,9 +102,13 @@ export class NotionImporter extends FormatImporter {
 					ctx.status(`Importing note ${fileInfo.title}`);
 
 					const markdownBody = await readToMarkdown(info, file);
+					const writeOptions: DataWriteOptions = {
+						ctime: fileInfo.ctime ? fileInfo.ctime.getTime() : Date.now(),
```

Comment:
I would prefer to not set this at all if we don't have this information (instead of forcefully setting it to `Date.now()`).

Reply:
The result of the current version of the behavior is that ctime and mtime will be the current time.